src/main/webapp/src/theme.scss

```diff
@@ -0,0 +1,198 @@
+/* Color scheme for our app -- primary #58d68d, secondary #005c80 */
+@import '~@angular/material/theming';
+
+// Theme generated by https://materialtheme.arcsine.dev/
+
+// Light Theme Text
+$dark-text: #000000;
```

Comment:
Why did you use scss instead of css. No problems with it, just wanted to know the difference :)

Reply:
I use SASS to specify the theme because you have to specify Material themes in this format. Of course I could compile the SASS down to CSS, but the generated CSS is minified and unreadable.

Reply:
Got it!